src/System.Management.Automation/utils/ClrFacade.cs

```diff
@@ -121,11 +105,7 @@ internal static T PtrToStructure<T>(IntPtr ptr)
             IntPtr ptr,
             bool deleteOld)
         {
-#if CORECLR
             Marshal.StructureToPtr<T>(structure, ptr, deleteOld);
-#else
-            Marshal.StructureToPtr(structure, ptr, deleteOld);
-#endif
```

Comment:
For instances like the above ones, we should make changes in the call sites of those methods to directly use the API available in .NET Core, so that we can completely remove those methods from ClrFacade.cs.